cmd/nerdctl/images_test.go

```diff
@@ -74,3 +75,20 @@ func TestImages(t *testing.T) {
 		return nil
 	})
 }
+
+func TestImagesFilter(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	tempName := testutil.Identifier(base.T)
+	base.Cmd("pull", testutil.CommonImage).AssertOK()
+
+	dockerfile := fmt.Sprintf(`FROM %s
```

Comment:
We should test with at least 2 / 3 images imo and also after filter if possible

Reply:
SGTM